# Supplementary Data and Scripts for "How many are you? Open data and bioinformatics reveal species misidentification and potential introgression in Chordodes (Phylum Nematomorpha)"



Supplementary data and scripts for "How many are you? Open data and bioinformatics reveal species misidentification and potential introgression in Chordodes (Phylum Nematomorpha)". The same scripts and data have been used for the presentation "Open data for understanding potential issues in species identification in non-model species: a case study with the genus Chordodes (Phylum Nematomorpha, Class Gordioida)", given at the Young Systematists' Forum 2025.

Available also at:

https://zenodo.org/records/18415953

Basically, there are all the scripts used for the study, plus some files useful for re-making the analyses and the mamba settings as txt files. It's only missing the Trinity files (too big), which are on Zenodo. Since some files can be around GBs in size, I could not upload everything. If you need more data, let me know.

Abstract:

The potential usage of genomic open data can help us to understand patterns in biodiversity. They can also be helpful for identifying morphologically similar species. An example of taxon in which this can be useful is Nematomorpha, one of the less studied animal phyla, for which data has started to be available recently and where species identification can be hard. In this study, I planned initially to evaluate the usage of mitochondrial data for population analyses using an RNA sequencing (RNA-seq) dataset labelled as belonging to Chordodes fukuii. After surprising results using extracted sequences from the barcoding gene cytochrome c oxidase subunit I (COXI), I evaluated species delimitation using a mix of a previously released double-digest restriction-site-associated DNA sequencing (ddRADseq) SRA dataset plus the RNA-seq one. PCA, R analyses through “adegenet” and ADMIXTURE confirmed the presence of two species in the RNA-seq dataset, which should be labelled as C. formosanus and C. japonensis; however, some individuals labelled as C. japonensis according to COXI clustered with C. formosanus‘s specimens or had some C. formosanus’ ancestry when more data was used, indicating potential introgression or incomplete lineage sorting. The study shows how previously released data can be used for evaluating species delimitation, potential previous demographic events and potential needs in DNA barcoding and genomics for avoiding future misidentification of morphologically similar species.
